sdks/java/testing/jpms-tests/build.gradle

```diff
@@ -31,13 +34,66 @@ enableJavaPerformanceTesting()
 description = "Apache Beam :: SDKs :: Java :: Testing :: JPMS Tests"
 ext.summary = "E2E test for Java 9 modules"
 
+/*
+ * List of runners to run post commit tests on.
+ */
+def postCommitRunnerClass = [
+        directRunner: "org.apache.beam.runners.direct.DirectRunner",
+        flinkRunner: "org.apache.beam.runners.flink.TestFlinkRunner",
+        dataflowRunner: "org.apache.beam.runners.dataflow.TestDataflowRunner",
+]
+for (String runner : postCommitRunnerClass.keySet()) {
```

Comment:
Aesthetically, I dislike building code identifiers (or similar) by string concat. Pragmatically it makes it hard to search for them. But looping out these configs makes sense. What do you think about something like a map from configuration name to runner class.

```
def postCommitConfigurations = {
   "directRunnerPostCommit" : "org.apache.beam.runners.direct.DirectRunner",
   "flinkRunnerPostCommit" : "org.apache.beam.runners.flink.TestFlinkRunner"
   "dataflowRunnerPostCommit" : "org.apache.beam.runners.dataflow.TestDataflowRunner"
}
```